ChangeLog

```diff
@@ -1,3 +1,10 @@
+2016-11-15 Nicole Kingsley <nbkingsley@ucdavis.edu>
+
+   * test_normalize_by_median.py, test_scripts.py:
```

Comment:
please put full path in -- e.g. `tests/{test_normalize_by_median.py,test_scripts.py}`